src/pages/index.js

```diff
@@ -109,43 +108,114 @@ addNewCardButton.addEventListener("click", () => {
 });
 
 function handleAddCardFormSubmit(data) {
-  const name = data.title;
-  const link = data.url;
-  renderCard({ name, link }, cardListEl);
-  newCardModal.close();
+  const name = data.name;
+  const link = data.link;
+  api
+    .newCardReq(name, link)
+    .then((result) => {
+      const { name, link } = result;
+      renderCard(result);
+
+      newCardModal.close();
+    })
+    .catch((err) => {
+      console.error(err);
+    });
 }
 newCardModal.setEventListeners();
 
+// Card Delete Modal Popup
+
+function handleCardDeleteClick(card) {
+  confirmationModal.open(card);
+}
+
 // --------------------------------------
 
 const editFormModal = new ModalWithForm(
   "#profile-edit-modal",
   handleProfileEditSubmit
 );
 
-
-
 profileEditButton.addEventListener("click", () => {
-  const { name, job } = userInfo.getUserInfo();
+  const { name, about } = userInfo.getUserInfo();
   profileEditForm.querySelector(".modal__input_type_title").value = name;
```

Comment:
**Could be improved**

`profileEditForm.querySelector(".modal__input_type_title")`
`profileEditForm.querySelector(".modal__input_type_description")`


It’s better to find **constants** only 1 time at the top of the file so as not to waste resources on searching them again and again when you call a method (function) because searching is a very **hard** operation for the browser engine